src/util/FactorUtil.js

```diff
@@ -143,11 +143,13 @@ define(['okta'], function (Okta) {
   };
 
   fn.getFactorLabel = function (provider, factorType) {
-    return factorData[fn.getFactorName(provider, factorType)].label;
+    var key = factorData[fn.getFactorName(provider, factorType)].label;
+    return Okta.loc(key, 'login');
```

Comment:
Yep, they are localized - I'm just changing the time that we evaluate the Okta.loc call.